dlp/src/test/java/com/example/dlp/InspectIT.java

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2017, Google, Inc. Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You may obtain a copy of the License
+ * at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.dlp;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.PrintStream;
+
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class InspectIT {
+  private ByteArrayOutputStream bout;
+  private PrintStream out;
+  private String bucket;
+  private String datastoreKind;
+
+  @Before
+  public void setUp() {
+    bout = new ByteArrayOutputStream();
+    out = new PrintStream(bout);
+    System.setOut(out);
+    assertNotNull(System.getenv("GOOGLE_APPLICATION_CREDENTIALS"));
+    assertNotNull(System.getenv("DLP_BUCKET_ID"));
```

Comment:
I added notes in the README.md on defining these. Do you have a better approach in mind ?